uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -197,6 +198,8 @@ define([
           canSubmit = canSubmit && !_.isEqual(initialLeaveRequestAttributes, this.request.attributes());
           // user has added a comment
           canSubmit = canSubmit || this.request.comments.length !== initialCommentsLength;
```

Comment:
What about this? If the logic is not correct either we need to fix or open new bug.